nx-X11/config/cf/lnxLib.rules

```diff
@@ -9,7 +9,7 @@
 #endif
 
 #ifndef UseRpath
-#define UseRpath NO
+#define UseRpath YES
```

Comment:
Yep, I failed to see that `ExtraLoadFlags` does *always* include `RpathLoadFlags` unconditionally, which may or may not be set based upon `UseRpath`, so that's actually fine.

Must have skipped `RpathLoadFlags` mentally and only saw the `-Wl,-rpath-link,...` part.